packages/runtime/datastore/src/dataStoreRuntime.ts

```diff
@@ -402,6 +403,21 @@ export class FluidDataStoreRuntime
 		return context.getChannel();
 	}
 
+	/**
+	 * Validate user provided channel ID
+	 * Channel ID has limitations. "/" is not allowed as IDs in storage can not have slashes - we parse tree paths and use "/" as separator.
+	 * "_" can't be used as it could result in collision of IDs with auto-assigned (by FF) short ID
+	 * @param id - channel ID.
+	 */
+	protected validateChannelId(id: string) {
+		if (id.includes("/")) {
+			throw new UsageError(`Id cannot contain slashes: ${id}`);
+		}
+		if (id.includes("_")) {
```

Comment:
This should be id.startswith("_") right? It can have underscores in between the names as that is pretty common in names.